runtime/queries/rust/highlights.scm

```diff
@@ -175,6 +181,10 @@
 
 "fn" @keyword.function
 
+"macro_rules!" @keyword.directive
```

Comment:
I also think this could use a better name, not sure if `macro_rules!` is a directive

Reply:
Macrorules is really just another macro (the fact that it currently isn't is an imolementation detail of the rust compiler) so it should be highlighted the same as any other macro in my opinion.